src/draw/sw/lv_draw_sw_private.h

```diff
@@ -31,14 +31,22 @@ extern "C" {
  *      TYPEDEFS
  **********************/
 
-struct _lv_draw_sw_unit_t {
-    lv_draw_unit_t base_unit;
+typedef struct {
     lv_draw_task_t * task_act;
```

Comment:
```suggestion
    lv_draw_task_t * volatile task_act;
```

It's worked fine all this time so approving anyways.